packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div
+    class="relative flex items-center px-4 py-2 transition text-tiny group"
+    :class="getBannerColor"
```

Comment:
```suggestion
    :class="bannerColor"
```

Let's avoid the `get` prefix since by default computed properties are getter only. If it was a method we could have kept the `get` prefix.